core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -2157,6 +2165,23 @@ private static RelDataType deriveTypeMapFromEntries(SqlOperatorBinding opBinding
           OperandTypes.NUMERIC,
           SqlFunctionCategory.NUMERIC);
 
+  /** The "LOG10(numeric)" function. Returns the base 10 logarithm of numeric. */
+  @LibraryOperator(libraries = {MYSQL, SPARK})
+  public static final SqlFunction LOG10_MS =
+      SqlBasicFunction.create("LOG10",
+          ReturnTypes.DOUBLE_FORCE_NULLABLE,
```

Comment:
ReturnTypes.DOUBLE_FORCE_NULLABLE does not take effect

Reply:
I changed DOUBLE_NULLABLE in log10 in SqlStdOperatorTable into ReturnTypes.DOUBLE_FORCE_NULLABLE, and it took effect. Should I modify this? I'm not sure.

Reply:
@tanclary Do you have any ideas? You can uncomment me and try to debug it. Thank you

Reply:
It may be a function loading conflict. You can change the SqlKind of the custom function,  instead of modifying the `SqlStdOperatorTable` function.